### ISSUE TITLE
Add better admin support/authentication.

### ISSUE THREAD

**Issue by Arctem** [Arctem/arcbot#13]

**Comment by Utanith** [comment]
Here's a note: You can check if a user is authenticated with nickserv via the WHO command, but NOT by using the filter to search for the +r flag that indicates a user is registered! If you WHO a user and they're authenticated with nickserv, the fifth (zero-index) item in the reply will contain the character 'r'. It also contains their idle status (H or G) and if they're op or something like that.

**Comment by Arctem** [comment]
Yeah, adding NickServ support is definitely an ideal way to make this work better. Currently I can define admins but there's nothing preventing someone from imitating an admin except for the fact that we're never offline.

**Comment by Arctem** [comment]
Tying this to NickServ is actually probably not a great choice since we might be out of luck if it's down. Will need to consider how closely it is tied.
